src/Features/LanguageServer/Protocol/Extensions/Extensions.cs

```diff
@@ -42,6 +42,16 @@ public static ImmutableArray<Document> GetDocuments(this Solution solution, Uri
             return documents;
         }
 
+        public static ImmutableArray<AnalyzerConfigDocument> GetAnalyzerConfigDocuments(this Solution solution, Uri documentUri)
```

Comment:
We will definitely need the changes in this file later - but I think for just the text sync handlers we don't need it.  Lets move it out into a different PR until we have code that actually exercises this